src/main/resources/bootstrap.properties

```diff
@@ -22,6 +22,11 @@ spring.cloud.config.watch.delay=1000
 spring.cloud.config.fail-fast=true
 health.config.enabled=true
 
+# https://apereo.github.io/cas/5.0.x/installation/Configuration-Properties.html#message-bundles
+# Allow to override messages locally via /var/opt/atlas/i18n/cas
+# In custom_messages only text messages to translate via crowdin
+cas.messageBundle.baseNames=file:///var/opt/atlas/i18n/cas/messages,file:///opt/atlas/i18n/cas/messages,classpath:other_messages,classpath:custom_messages,classpath:messages
```

Comment:
This should be in `application.yml` instead of `bootstrap.properties` (`bootstrap.properties` should probably be removed from our overlay)

Reply:
Done. Thanks for the review.